Improve Android launch commands

sargant opened the pull request:
I have tested Pegasus on my Retroid Pocket 2+, running Android 9, and I have simplified the Android launch commands to make them less confusing to newcomers.

* The `IME`, `DATADIR`, `APK`, `SDCARD`, and `EXTERNAL` flags are all non-essential
* `--activity-single-top` ensures there is only one RetroArch task (it doesn't seem to load new tasks with different intents anyway)
* `QUITFOCUS` means that if another app is brought into the foreground, RetroArch quits. This is particularly useful if your Android device has a home button, as it means the game is quit properly when a new game is launched from Pegasus, and it doesn't simply resume the old game.

mmatyas commented:
Interesting! Could you point me where `QUITFOCUS` is used in RetroArch? I've based the launch command on its code at the time of writing – I'm not sure if there's any core that still uses the fields, but since RA itself passes them too, I fear we'd run into some corner cases with some of them if they aren't there when needed.

Also just to make sure, at the moment all three of

```
  --activity-clear-task
  --activity-clear-top
  --activity-no-history
```

are added to the commands by the metadata generator page; perhaps the documentation could use all of them too.

sargant commented:
`QUITFOCUS` was introduced in this PR: https://github.com/libretro/RetroArch/pull/4619

And this is the file it targets: https://github.com/libretro/RetroArch/blob/master/pkg/android/phoenix/src/com/retroarch/browser/retroactivity/RetroActivityFuture.java

As for the fields that I've removed, it seems like sensible defaults are provided for most of them in the RetroArch source code for the Android app right here: https://github.com/libretro/RetroArch/blob/master/pkg/android/phoenix/src/com/retroarch/browser/mainmenu/MainMenuActivity.java#L160

As for the `--activity-*` flags, I've been testing on my RP2+ and I can't tell if any of them have any effect at all, even the new one I added. Can do some more testing on those.

mmatyas commented:
Checked the sources, the RA flags parts looks good! For the activity options, the ones I listed above seem to resolve issues in certain apps, so let's use those.

mmatyas commented:
Checked the sources, the RA flags parts looks good! For the activity options, the ones I listed above seem to resolve issues in certain apps, so let's use those.

sargant commented:
Great stuff - I've updated the PR to put `--activity-clear-top` back in instead of `--activity-single-top`, and added the two newer ones.

mmatyas commented:
Thanks!

mmatyas commented:
I'll update the actual site later because the tools are not in the mood to get work done today.
